news/2024/2024-07-06-osu-catch-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: osu!catch World Cup 2024 Concludes
+date: 2024-07-06 12:00:00 +0000
+---
+
+After years of South Korean domination, this year, we finally crown a new osu!catch World Cup champion.
```

Comment:
"this year" feels redundant, i feel like it should either be removed or edited like below
```suggestion
After years of South Korean domination, this year we finally crown a new osu!catch World Cup champion.
```